index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="styles.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to my personal portfolio</title>
+
+</head>
+
+<body>
+    <header id="about">
+        <nav id="navbar">
+            <ul>
+                <li><a href="#about">About</a></li>
+                <li><a href="#work">Work</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+
+        </nav>
+        <div id="welcome-section" class="container"></div>
+        <h1 id="title">Duncan Benjamin</h1><a href="https://github.com/osbentech/Benjamin-Duncan-project.git">visit my porfolio</a>
+        <p class="lead"><em>A web developer</em></p>
+        <div id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="duncan benjamin">
+        </div>
+    </header>
+    <section id="projects">
```

Comment:
nice job,
over here I noticed that there is no section closing tag

Reply:
- [ ] Please try to state the reason why you are requesting changes to the students to make them aware of the issues with their code and the importance of modifying it in that way they will be more convinced to make changes 💯 . feel free to make all your comments below more descriptive by stating why you request changes.